### PR TITLE
deps: bump kubebuilder, align ansible/helm plugins with `kustomize/v1` updates

### PR DESCRIPTION
**Pull request by estroz**
**Description of the change:**
- go.mod,go.sum: bump kubebuilder
- internal/plugins: align ansible/helm plugins with kustomize/v1 updates

**Motivation for the change:** kubebuilder changes

/area dependency

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: internal/plugins/util/cleanup.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+// RemoveKustomizeCRDWebhooks removes items in config/crd relating to CRD conversion webhooks.
+func RemoveKustomizeCRDWebhooks() error {
+
+	if err := util.ReplaceInFile(filepath.Join("config", "crd", "kustomization.yaml"),
+		`
+patchesStrategicMerge:
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
+# patches here are for enabling the conversion webhook for each CRD
+#- patches/webhook_in_memcacheds.yaml
```
Comment (estroz) [inline internal/plugins/util/cleanup.go:35]: This won't quite work. Need to figure out a better way to overwrite this `kustomization.yaml`.

**Review by camilamacedo86 — APPROVED** [review]
That shows great 🥇 We could simplify more yet.
Now the Ansible/Helm plugins have more granular yet what is specific for them.

**Comment by fabianvf** [comment]
/lgtm

> +218 −574 

You love to see it

**Pull request merged**